storage/eoswrapper/eoswrapper.go

```diff
@@ -234,7 +235,37 @@ func (w *wrapper) userIsProjectAdmin(ctx context.Context, ref *provider.Referenc
 		// Nothing to do in that case
 		return nil
 	}
-	adminGroup := projectSpaceGroupsPrefix + parts[2] + projectSpaceAdminGroupsSuffix
+	adminGroup := projectSpaceGroupsPrefix + parts[2] + projectSpaceAdminsGroupSuffix
+	user := appctx.ContextMustGetUser(ctx)
+
+	for _, g := range user.Groups {
+		if g == adminGroup {
+			return nil
+		}
+	}
+
+	return errtypes.PermissionDenied("eosfs: project spaces revisions can only be accessed by admins")
+}
+
+func (w *wrapper) userIsProjectWriter(ctx context.Context, ref *provider.Reference) error {
+	// Check if this storage provider corresponds to a project spaces instance
+	if !strings.HasPrefix(w.conf.Namespace, eosProjectsNamespace) {
+		return nil
+	}
+
+	res, err := w.GetMD(ctx, ref, nil)
+	if err != nil {
+		return err
+	}
+
+	// Extract project name from the path resembling /c/cernbox or /c/cernbox/minutes/..
+	parts := strings.SplitN(res.Path, "/", 4)
+	if len(parts) != 4 && len(parts) != 3 {
+		// The request might be for / or /$letter
+		// Nothing to do in that case
+		return nil
+	}
+	adminGroup := projectSpaceGroupsPrefix + parts[2] + projectSpaceWritersGroupSuffix
```

Comment:
```suggestion
	writerGroup := projectSpaceGroupsPrefix + parts[2] + projectSpaceWritersGroupSuffix
```

Reply:
(that's cosmetics but just for clarity - and the reference in L272 has to be fixed accordingly)